Framework/Algorithms/src/MergeRuns/SampleLogsBehaviour.cpp

```diff
@@ -371,6 +376,22 @@ void SampleLogsBehaviour::mergeSampleLogs(MatrixWorkspace &addeeWS,
 }
 
 /**
+ * Perform the update for a sum property, adding a new value to the existing
+ *one. Skipped if the time series log entry is in the addeeWS.
+ *
+ * @param addeeWSNumber the sample log value from the workspace being merged
+ * @param outWSNumber the sample log value for the merged workspace
+ * @param outWS the workspace the others are merged into
+ * @param name the name of the property
+ */
+void SampleLogsBehaviour::updateSumProperty(double addeeWSNumber,
+                                            double outWSNumber,
+                                            MatrixWorkspace &outWS,
+                                            const std::string name) {
+  outWS.mutableRun().addProperty(name, addeeWSNumber + outWSNumber, name, true);
```

Comment:
Why `name` is here twice? Isn't this the
```
addProperty(const std::string &name, const TYPE &value,
                   const std::string &units, bool overwrite = false)
```
signature, in which case you are giving quite a strange units for the property.